platform/broadcom/sai.mk

```diff
@@ -1,6 +1,6 @@
 LIBSAIBCM_XGS_VERSION = 10.1.37.0
 LIBSAIBCM_DNX_VERSION = 10.1.25.0
-LIBSAIBCM_XGS_BRANCH_NAME = SAI_10.1.0_GA_CS00012362288
+LIBSAIBCM_XGS_BRANCH_NAME = SAI_10.1.0_GA
```

Comment:
Please update 202405 branch as well.

I will close this 202405 PR - https://github.com/sonic-net/sonic-buildimage/pull/19839

Reply:
We see cherrypick conflict for 202311 and 202405 branches for sai.mk, could you check so that we can avoid manual sync?

Reply:
dnx SAI version is not same in master, 202311, and 202405 branch, the conflict is expected. I'll create PR to 202311 and 202405 branch manually for now.